demo/gpu-test6.yaml

```diff
@@ -0,0 +1,48 @@
+# One pod: one initContainer + one container
+# Each asking for shared access to a single GPU
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gpu-test6
```

Comment:
@klueska Is there any source material for this image where we could integrate this new example? https://github.com/kubernetes-sigs/dra-example-driver/blob/4a1bf91c32d7ccf43e93d7cd6dfaaf7564a7fae2/demo/demo-apps.png